test/CMakeLists.txt

```diff
@@ -102,21 +105,21 @@ addLinkAndDiscoverTest(IndexMetaDataTest index)
 # Stxxl currently always uses a file ./-stxxl.disk for all indices, which
 # makes it impossible to run the test cases for the Index class in parallel.
 # TODO<qup42, joka921> fix this
-addLinkAndDiscoverTestSerial(IndexTest index)
+addLinkAndDiscoverTestSerial(IndexTest index testUtil)
```

Comment:
Would it work, to automatically link this library against ALL tests, e.g. by hardcoding it
into the various `addLink...` functions above?
This would make life easier, and we probably need them more often, and we break less code if we move more of those helpers to `.cpp` files.

Reply:
If you think, it's wise. Done in [0435515](https://github.com/ad-freiburg/qlever/pull/1016/commits/0435515fae49f2a2cd39de6826e80b400ff4f0a9).